client/dist/index.html

```diff
@@ -2,10 +2,10 @@
 <html>
   <head>
     <title>Summary Bar</title>
-    <link rel="stylesheet" href="style.css">
+    <!-- <link rel="stylesheet" href="style.css"> -->
```

Comment:
@rgagnon2 curious about the html page without a stylesheet reference.
Assuming it is going to be included later, it could be a good choice to either have the code uncommented or remove it